docs/provider/daemon/getting_started/run_remote.md

```diff
@@ -0,0 +1,131 @@
+
+## Enable macaroons/TLS for Operator interface on remote host
+
+If you want to run a daemon on a remote machine and make its Operator interface reachable from the outside world it's strongly suggested to enable macaroons auth and TLS encryption to secure the connection with external clients. 
+
+You must have a static IP or a DNS name at which to reach your machine and configure it to allow in-going and out-going traffic over the Operator and Trade ports (the defaults are `9000` and `9945` respctively).
+
+Before starting the daemon, export the environment variables `TDEX_OPERATOR_EXTRA_IP` and/or `TDEX_OPERATOR_EXTRA_DOMAIN` like for example:
+
+- Standalone:
+    ```sh
+    $ export TDEX_OPERATOR_EXTRA_IP=54.170.20.39
+    $ export TDEX_OPERATOR_EXTRA_DOMAIN=provider.mydomain.com
+
+    # Start a daemon with macaroons/TLS enabled ONLY on Operator interface
+    tdexd
+    ```
+
+- Docker:
+  ```sh
+  # Start a dockerized daemon with macaroons/TLS enabled ONLY on Operator interface
+  $ docker run -it -d \
+    --name tdexd \
+    --restart unless-stopped \
+    -p 9945:9945 -p 9000:9000 \
+    -v `pwd`/tdexd:/.tdex-daemon \
+    -e TDEX_OPERATOR_EXTRA_IP=54.170.20.39 \
+    -e TDEX_OPERATOR_EXTRA_DOMAIN=provider.mydomain.com \
+    ghcr.io/tdex-network/tdexd:latest
+  ```
+
+This will make the daemon create the TLS certificate by including the given IP/DNS name to the list of identities, making it usable for external clients.
+
+## Enable TLS for Trade interface on remote host
+
+It is possible to enable TLS encryption and requiring to establish a secure connection with the public Trade interface of the daemon.
+
+For this, you must obtain a valid TLS certificate from a trusted Certificate Authotity (for example [Let's Encrypt](https://letsencrypt.org/), [ZeroSSL](https://zerossl.com/), etc.) and export the `TDEX_SSL_KEY` and `TDEX_SSL_CERT` environment variables with the paths of your TLS key and cert files:
+
+- Standalone:
+    ```sh
+    # In this example the TLS key/cert were previously generated with the help of
+    # Let's Encrypt and placed in a dedicated directory into the daemon datadir
+    $ export TDEX_SSL_KEY=~/.tdex-daemon/trade-tls/privatekey.pem
+    $ export TDEX_SSL_CERT=~/.tdex-daemon/trade-tls/fullchain.pem
+
+    # Start a daemon with TLS enabled ONLY on Trade interface
+    $ tdexd
+    ```
+- Docker:
+    ```sh
+    # Start a dockerized daemon with TLS enabled ONLY on Trade interface
+    $ docker run -it -d \
+    --name tdexd \
+    --restart unless-stopped \
+    -p 9945:9945 -p 9000:9000 \
+    -v `pwd`/tdexd:/.tdex-daemon \
+    # Here, it's required to mount the filepaths of the TLS key/cert as volumes
+    # of the container
+    -v `pwd`/privatekey.pem:/privatekey.pem \
+    -v `pwd`/fullchain.pem:/fullchain.pem \
+    -e TDEX_SSL_KEY=/privatekey.pem \
+    -e TDEX_SSL_CERT=/fullchain.pem \
+    ghcr.io/tdex-network/tdexd:latest
+    ```
+
+:::tip
+Enable secure connection on both interfaces
```

Comment:
here we can have a second way, which is onion hidden service, not only TLS server side